doc/manual/source/parameters/index.rst

```diff
@@ -3438,7 +3440,20 @@ Scale-similarity model
     Recommended: 0.89;
 
 
+.. _fdm_parameters:
```

Comment:
Need to add the FDM problem types to the `ProblemType` list and descriptions of the different `LightBosonProblemType` choices, which needs to be turned into a parameter.  It's hard-coded to be 1 right now.

Also, I think there needs to be a brief description of how to set up a FDM cosmology run with running the provided script beforehand.

Reply:
The LightBosonProblemType is now described (the parameter is not hard-coded now). 

I agree that it would be great to have a description of how to set up an FDM cosmology, but this is a bit beyond this pull request.  I've made the lack of a mechanism to generate a cosmological initial wavefunction clearer in the description.